src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -484,22 +483,22 @@ public void addOnRankedMatchNotificationListener(Consumer<MatchmakerInfoMessage>
     fafService.addOnMessageListener(MatchmakerInfoMessage.class, listener);
   }
 
-  public CompletableFuture<Void> startSearchLadder1v1(Faction faction) {
+  public CompletableFuture<Void> startSearchLadder(RatingMode ratingMode, Faction faction) {
```

Comment:
Maybe it's better to take both rating mode and game mode as arguments here. On the TMM branch both of these parameters will come from the matchmaker queue object.

Reply:
Why do we need game mode here? This method is dedicated to launching ladder games only.

Reply:
When I say game mode I mean “featured mod”, and matchmaker games can be played on any featured mod.